pkg/arvo/app/dojo.hoon

```diff
@@ -1019,6 +1019,8 @@
           ==
         ::
             $listen-api  !!
+            $export      !!
+            $import      !!
```

Comment:
The existence of these suggests that layering between `:lens` and `:dojo` is wrong. (This was already kind of obvious, but now it's more so.) Nothing needs to be done about it now, but I think a fair amount of the lens command parsing/handling should move from `:dojo` to `:lens`. Just noting for posterity.